src/views/admin/PersonDetailsView.vue

```diff
@@ -1879,7 +1881,7 @@
                       {{ $t('person.addZuordnung') }}
                     </v-btn>
                   </SpshTooltip>
-                  <!-- This will stay commented until the buttons are actually functionable 
+                  <template v-if="configStore.configData?.rolleBearbeitenEnabled">
```

Comment:
Verständnisfrage: Bleibt das für immer erhalten oder wird die Condition irgendwann wieder ausgebaut?

Reply:
Weiss Ich auch nicht ganz genau. 

Wir wollten erstmal den neuen Endpunkt testen, in dem wir die unbenutzten 2 Buttons hier aus/einblenden abhängig vom Flag. Wenn wir mit der Implementierung der Funktion für die Buttons anfangen dann wird die condition entfernt oder der Flag im Backend auf True gesetzt werden.